demos/common/python/openvino/model_zoo/model_api/adapters/utils.py

```diff
@@ -0,0 +1,76 @@
+"""
+ Copyright (C) 2022 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import re
+from typing import Optional
+from openvino.runtime import layout_helpers
+
+
+class Layout:
+    def __init__(self, layout = '') -> None:
+        self.layout = layout
+
+    @classmethod
+    def from_shape(cls, shape):
+        '''
+        Create Layout from given shape
+        '''
+        if len(shape) != 4:
+            raise RuntimeError('Get layout from shape method supports only 4D input shape')
+
+        layout = 'NCHW' if shape[1] in range(1, 4) else 'NHWC'
+        return cls(layout)
+
+    @classmethod
+    def from_openvino(cls, input):
+        '''
+        Create Layout from openvino input
+        '''
+        return cls(layout_helpers.get_layout(input).to_string().strip('[]').replace(',', ''))
```

Comment:
We always take `layout` attribute from the class instance in `get_layout_for_input()`. Are there any pros to return cls instead of string representation itself? If no, please return layout instead in methods
 ```suggestion
         return layout_helpers.get_layout(input).to_string().strip('[]').replace(',', '')
```